templates/support/index.html

```diff
@@ -1,151 +1,189 @@
-                    <img class="inline-logos__image" src="{{ ASSET_SERVER_URL }}0c18b0ae-paypal_logo.svg" width="166" alt="Paypal logo">
-                </li>
-            </ul>
-        </div>
-    </div>
+<section class="p-strip--x-light is-bordered is-deep">
+<div class="row">
+  <div class="ten-col push-one align-center">
+    <h2 class="p-heading--muted u-align--center">A selection of Ubuntu Advantage customers</h2>
+    <ul class="p-inline-images">
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}1a062141-logo-bloomberg.svg" width="187" alt="Bloomburg logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}3c7e83fc-atandt-logo.svg" width="61" alt="at&amp;t  logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}3161e6ba-logo-walmart.svg" width="210" alt="Walmart logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}03a06060-logo-deutschetelekom.svg" width="242" alt="Deutshe Telekom logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}e0f7037f-logo-ebay.svg" width="170" alt="Ebay logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}06c5609f-logo-cisco.svg" width="156" alt="Cisco logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}ddb0f9d6-logo-ntt.svg" width="71" alt="NTT logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}15e9c496-logo-bestbuy.svg" width="166" alt="Best Buy logo">
+      </li>
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}0c18b0ae-paypal_logo.svg" width="166" alt="Paypal logo">
+      </li>
+    </ul>
+  </div>
+</div>
 </section>
 
-<section id="community-support" class="row strip-light no-border">
-    <div class="strip-inner-wrapper">
-        <div class="eight-col">
-            <h2>Community support</h2>
-            <p>As one of the largest open source projects, with Ubuntu you can find support from a variety of sources. Take a look - you&rsquo;re likely to find an answer to every question. If you can&rsquo;t find an answer, just ask the people in our active forums.</p>
-            <p><a href="/support/community-support">Learn more about community support&nbsp;&rsaquo;</a></p>
-        </div>
-        <div class="four-col last-col text-center priority-0">
-            <img src="{{ ASSET_SERVER_URL }}039628d5-picto-community-orange.svg" width="150" alt="Community">
-        </div>
+<section id="community-support" class="p-strip is-deep">
```

Comment:
p-strip--light according to https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/8.%20Support?preview=support-overview.png